javascript/atoms/action.js

```diff
@@ -87,21 +87,25 @@ bot.action.clear = function(element) {
         'Element must be user-editable in order to clear it.');
   }
 
-  bot.action.LegacyDevice_.focusOnElement(element);
```

Comment:
Both branches appear to call `focusOnElement` as the first thing they do. Why not just leave this up here?

Reply:
Because the spec specifies that the element not be focused unless it has a value to be cleared. Leaving this outside the branches would cause the element to be focused indiscriminately.